src/app/services/single-page/get-single-hero.service.ts

```diff
@@ -0,0 +1,31 @@
+import { HeroModel } from './../../models/heroModel';
+import { environment } from './../../../environments/environment';
+import { IHero } from './../../interface/heroInterface';
+import { Observable, throwError } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { map, catchError } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GetSingleHeroService {
```

Comment:
As above, I think we should just call this HeroService, so in the future we could just add hero related services to this class, rather than creating a new one for every similar hero related task?

Reply:
Same as earier